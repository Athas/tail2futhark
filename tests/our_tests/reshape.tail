let v1:[int]2 = reshape{[int],[1,2]}([3,3],[4,5]) in
i2d(reduce{[int],[0]}(addi,0,shape{[int],[2]}(v1)))
