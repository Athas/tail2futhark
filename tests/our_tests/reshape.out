fun real main() =
  let t_v1 = reshape((3,3),reshape1int(3 * 3 * 1,reshape((size(0,[4,5]) * 1),[4,5]))) in
  toReal(reduce(+,0,[size(0,t_v1),size(1,t_v1)]))